__init__.py

```diff
@@ -31,5 +31,6 @@ def demangle_swift(bv):
 	assert len(swift_variables) == len(results)
 	for (variable, name) in zip(swift_variables, results):
 		variable.name = name
+	binaryninja.log_info("Swift demangling complete")
```

Comment:
```suggestion
	binaryninja.log_info("Swift demangling complete!")
```

If you want to go above and beyond I'd also add a bit of detail about how much work was done, e.g.

```suggestion
	binaryninja.log_info(f"Swift demangling complete! Updated {len(swift_functions)} functions and {len(swift_variables)} variables.")
```

Reply:
That also just seems like potentially useful information, good idea